dhis-2/dhis-api/src/main/java/org/hisp/dhis/tracker/imports/validation/ValidationCode.java

```diff
@@ -157,6 +157,7 @@ public enum ValidationCode {
   E1312("Referral events need to have both sides of a relationship."),
   E1313(
       "Event {0} of an enrollment does not point to an existing tracked entity. The data in your system might be corrupted"),
+  E1316("Event `{0}` cannot transition from status `{1}` to status `{2}.`"),
```

Comment:
Why is there a gap in error codes?

Should we phrase it as `No event can transition...` as its not specific to that event. The uid of the event is returned in the object containing the message right?